plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduClientConfig.java

```diff
@@ -32,14 +33,14 @@
 /**
  * Configuration read from etc/catalog/kudu.properties
  */
+@DefunctConfig("kudu.client.default-socket-read-timeout")
```

Comment:
reword commit message from `Deprecate` to `Remove` - since deprecation doesn't cause failures for end users and doesn't require changes but this will require changes.